src/Cli/dotnet/commands/dotnet-workload/install/NetSdkManagedInstaller.cs

```diff
@@ -114,7 +117,16 @@ public void InstallWorkloadPack(PackInfo packInfo, SdkFeatureBand sdkFeatureBand
                         WritePackInstallationRecord(packInfo, sdkFeatureBand);
                     },
                     rollback: () => {
-                        RollBackWorkloadPackInstall(packInfo, sdkFeatureBand);
+                        try
+                        {
+                            _reporter.WriteLine(string.Format(LocalizableStrings.RollingBackPackInstall, packInfo.Id));
+                            RollBackWorkloadPackInstall(packInfo, sdkFeatureBand);
+                        }
+                        catch (Exception e)
+                        {
+                            // Don't hide the original error if roll back fails
+                            _reporter.WriteLine(string.Format(LocalizableStrings.RollBackFailedMessage, e.Message));
```

Comment:
Writing the `Message` property of an exception is often not very helpful.  I'm not sure there's a better option here though.  `ToString()` is generally better for diagnostic logs, but it includes the stack trace so that may be too verbose for here.